qemu/tests/cfg/qemu_disk_img.cfg

```diff
@@ -60,26 +62,66 @@
                                     cluster_size = 2M
                         - compat:
                             no Host_RHEL.m5, Host_RHEL.m6
+                            option_verified += " compat"
                             variants:
```

Comment:
why not 
variants:
- compat_0.10:
- compat_1.1:
- variants:
 - default:
....